packages/db-dashboard/index.js

```diff
@@ -80,25 +80,30 @@ function transformHttpPromMetrics (httpMetrics = []) {
 
 module.exports = async function app (app, opts) {
   app.log.info('dashboard plugin loaded.')
-  if (opts.dashboardAtRoot !== false) {
+  let dashboardPath = '/dashboard'
+  
+  if (opts.path) {
+    dashboardPath = opts.path
+  } else {
     app.get('/', { hide: true }, function (req, reply) {
-      return reply.redirect(302, '/dashboard')
+      return reply.redirect(302, dashboardPath)
     })
   }
 
   app.register(fastifyStatic, {
-    root: path.join(__dirname, 'build')
+    root: path.join(__dirname, 'build'),
+    prefix: dashboardPath
   })
 
-  app.get('/dashboard', { hide: true }, function (req, reply) {
+  app.get(`${dashboardPath}`, { hide: true }, function (req, reply) {
     return reply.sendFile('index.html')
   })
 
-  app.get('/dashboard/*', { hide: true }, function (req, reply) {
-    return reply.sendFile('index.html')
-  })
+  // app.get(`${dashboardPath}/*`, { hide: true }, function (req, reply) {
+  //   return reply.sendFile('index.html')
+  // })
```

Comment:
why is this commented out?

Reply:
`fastify-static` gives me the error of #560 , I'm looking how to solve this